### ISSUE TITLE
Rename files

### ISSUE THREAD

**Issue by xthephreakx** [oae/kaizoku#73]
### Feature Description

Is there a way to rename the files after downloading? Similar to how Mylar does this ?

### Solution you would like

Get the name from the meta data?

**Comment by oae** [comment]
Most of the manga in anilist doesn't have any chapter names.  So I don't know how it can be useful

**Comment by xthephreakx** [comment]
True was more looking at just adding the name of the manga.  

Example dragon ball super is called: [0001]_Chapter_1.cbz

But full metal alchemist is also called : [0001]_Chapter_1.cbz

So when adding these for example to my comic reader (chunky comic reader iOS) it sees the same name so doesn't no how to group it. 

But if it is possible to add at least the name of the manga behind [0001] the problem would be solved.

**Comment by oae** [comment]
All downloaded chapters contain metadata inside `ComicInfo.xml`. This file also includes the title and series name. If your reader supports getting metadata from this file, it shouldn't be a problem. The current architecture of the Kaizoku doesn't allow renaming chapters.

```xml
<ComicInfo xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Title>Spell 47</Title>
  <Series>Dorohedoro</Series>
  <Number>54</Number>
  <Web>https://mangasee123.com/read-online/Dorohedoro-chapter-47-page-1.html</Web>
  <Genre>Action,Adventure,Comedy,Fantasy,Horror,Mystery</Genre>
  <PageCount>24</PageCount>
  <Summary>In a city so dismal it&#39;s known only as &#34;the Hole,&#34; a clan of Sorcerers have been plucking people off the streets to use as guinea pigs for atrocious &#34;experiments&#34; in the black arts. In a dark alley, Nikaido found Caiman, a man with a reptile head and a bad case of amnesia. To undo the spell, they&#39;re hunting and killing the Sorcerers in the Hole, hoping that eventually they&#39;ll kill the right one. But when En, the head Sorcerer, gets word of a lizard-man slaughtering his people, he sends a crew of &#34;cleaners&#34; into the Hole, igniting a war between two worlds.&#xA;&#xA;&#xA;&#xA;(Source: VIZ Media)&#xA;&#xA;&#xA;&#xA;Note: Includes 23 side stories titled &#34;Extra Evil&#34; and the side story &#34;The Lizard Head and the Magic Whistle.&#34;</Summary>
  <Count>191</Count>
  <Characters>Risu,Nikaido,Shin,Noi,En,Fujita,Aikawa,Kai,Ebisu,Kaiman</Characters>
  <Year>2023</Year>
  <Month>1</Month>
  <Day>12</Day>
  <Writer>Q Hayashida</Writer>
  <Translator>Matt Alt,Hiroko Yoda,Gabriel Álvarez Martínez,Dariusz Latoś</Translator>
  <Tags>Dissociative Identities,Magic,Ensemble Cast,Anti-Hero,Amnesia,Seinen,Revenge,Body Horror,Gore,Primarily Adult Cast,Surreal Comedy,Demons,Time Manipulation,Super Power,Monster Boy,Tragedy,Urban Fantasy,Body Swapping</Tags>
  <Notes>Downloaded with Mangal. https://github.com/metafates/mangal</Notes>
  <Manga>YesAndRightToLeft</Manga>
</ComicInfo>
```

**Comment by xthephreakx** [comment]
Yeah.I noticed that aswell that's why komga had no issue in displaying the names only "issue" is on the iPad when I download them but no worries I'll try and make a workaround for it

**Comment by GlassedSilver** [comment]
Another useful add would be to mark volumes for chapter ranges.

%MANGA% - Vol. %VOL% - Ch. %CHAPTER% - %CHAPTERNAME% [%SCANLATIONGROUPandEDIT/"FANCOLORED"/OTHERDIFFERENTIATORS].%ext%

This would be a banger. FMD2 and pulling from Mangadex comes veeeery close, but if volume isn't supplied by Mangadex you can't define volume ranges manually and hence need to rename by hand in the file manager, a bit annoying for mobile work flows and very reactive rather than proactive. :(

**Comment by IsXGeo** [comment]
> Another useful add would be to mark volumes for chapter ranges.
> 
> %MANGA% - Vol. %VOL% - Ch. %CHAPTER% - %CHAPTERNAME% [%SCANLATIONGROUPandEDIT/"FANCOLORED"/OTHERDIFFERENTIATORS].%ext%
> 
> This would be a banger. FMD2 and pulling from Mangadex comes veeeery close, but if volume isn't supplied by Mangadex you can't define volume ranges manually and hence need to rename by hand in the file manager, a bit annoying for mobile work flows and very reactive rather than proactive. :(
> 
> Also, that means the info is missing from the metadata file.
> 
> Kavita would be very happy to have this information, too.

I think this would be a useful feature, since some chapters filenames can also not be consistent. 
Four examples from "The Gamer":
[0160]_Chapter_158_The_Gamer_S003_-_Chapter_035.cbz
[0188]_Chapter_186.cbz
[0321]_Chapter_319_[Season_4]_Ep._124.cbz
[0030]_Chapter_29_Experience_Determines_One_S_Life.cbz

In my case, I'd like to get rid of season information since Kavita treats "seasons" as entirely different series and manually renaming a bunch of chapters is not a fun task.
